gateway/api/v1/views.py

```diff
@@ -69,13 +70,37 @@ class JobViewSet(views.JobViewSet):  # pylint: disable=too-many-ancestors
     Job view set first version. Use JobSerializer V1.
     """
 
-    queryset = Job.objects.all()
     serializer_class = v1_serializers.JobSerializer
+    pagination_class = LimitOffsetPagination
     permission_classes = [permissions.IsAuthenticated, IsOwner]
 
     def get_serializer_class(self):
         return v1_serializers.JobSerializer
 
+    @swagger_auto_schema(
+        operation_description="Get author Job",
+        responses={status.HTTP_200_OK: v1_serializers.JobSerializer(many=False)},
+    )
+    def retrieve(self, request, pk=None):
+        return super().retrieve(request, pk)
+
+    @swagger_auto_schema(
+        operation_description="List author Jobs",
+        responses={status.HTTP_200_OK: v1_serializers.JobSerializer(many=True)},
+    )
+    def list(self, request):
+        return super().list(request)
+
+    @swagger_auto_schema(
+        operation_description="Save the result of a job",
+        responses={status.HTTP_200_OK: v1_serializers.JobSerializer(many=False)},
+    )
+    @action(methods=["POST"], detail=True)
+    def result(self, request, pk=None):
+        return super().result(request, pk)
+
+    ### We are not returning serializers in the rest of the end-points
```

Comment:
We have a problem here because in the rest of the end-points we are not using serializers to format the response. I will open an issue for this, fix it would do the PR so big 😅